Whiskey/Tasks/Invoke-WhiskeyPowerShell.ps1

```diff
@@ -1,7 +1,66 @@
 
 function Invoke-WhiskeyPowerShell
 {
-    [Whiskey.Task("PowerShell")]
+    <#
+    .SYNOPSIS
+    Executes PowerShell tasks.
+
+    .DESCRIPTION
+    The PowerShell tasks runs PowerShell scripts. You specify the scripts to run via the `Path` property. Paths must be relative to the whiskey.yml file. Pass arguments to the scripts with the `Argument` property, which is a hash table of parameter names and values. PowerShell scripts are run in new, background processes.
+
+    The PowerShell task runs your script in *all* build modes: during builds, during initialization, and during clean. If you want your script to only run in one mode, use the global `OnlyDuring` property to specify the mode you want it to run in or the `ExceptDuring` property to specify the run mode you don't want it to run in.
+
+    The PowerShell task will fail a build if the script it runs returns a non-zero exit code or sets the `$?` variable to `$false`.
+
+    To receive the current build context as a parameter to your PowerShell script, add a `$TaskContext` parameter, e.g.
+
+        param(
+            [object]
+            $TaskContext
+        )
+
+    This is *not* recommended.
+
+    ## Properties
+    * **Path** (mandatory): the paths to the PowerShell scripts to run. Paths must be relative to the  whiskey.yml file. Script arguments are not supported.
+    * **WorkingDirectory**: the directory from which the scripts should be run. This path can be absolute. The default is the  whiskey.yml file's directory.

```

Comment:
Isn't `WorkingDirectory` a global property now? Are there certain global properties we always include in task docs?